sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -671,7 +671,7 @@ pi_result _pi_device::initialize(int SubSubDeviceOrdinal,
   // At present even PVC doesn't automatically use immediate commandlists.
   // Change this after more testing.
   ImmCommandListsPreferred =
-      false; // (ZeDeviceProperties->deviceId & 0xff0) == 0xbd0;
+      true; // (ZeDeviceProperties->deviceId & 0xff0) == 0xbd0;
```

Comment:
do we need this comment still, indicating we need to check for 0bd0? if this is for all devices, this comment seems not needed anymore.

Reply:
It is an open question whether to switch to immediate command lists for all GPUs or only for the PVC family. L0 and Driver input on this would be welcome.

Reply:
@rdeodhar : we should change by default to all, that's what L0 GPU driver has been working on. But that would depend also on whether we have SYCL CI results with other gens, do we have that?

Reply:
Testing in CI is ongoing.

Reply:
@rdeodhar does setting this to true enable immediate commandlists for CCS only or BCS as well? (ideally we'd want both)

Reply:
It does both CCS and BCS.